src/engine/Debug.ts

```diff
@@ -292,6 +306,7 @@ export class FrameStats implements IFrameStats {
 export class PhysicsStats implements IPhysicsStats {
    private _pairs: number = 0;
    private _collisions: number = 0;
+   private _collidersHash: ICollidersHash = {};
```

Comment:
I like typing the hash with an interface 👍 probably going to start doing this in the future especially on intereseting type signatures :)